orchestrator/tests/test_prd224_assign_lane.py

```
@@ -0,0 +1,244 @@
+
+
+# ---------------------------------------------------------------------------
+# roster name-matching → target_agent_id (assessment stubbed)
+# ---------------------------------------------------------------------------
+
+
+def _brain():
+    return AutoBrain(None, "00000000-0000-0000-0000-000000000001")
+
+
+def _agent(id, name):
```

Ruff (0.16.2)

[error] Function argument `id` is shadowing a Python builtin

(A002)